users/pharnb/index.md

```diff
@@ -6,10 +6,30 @@ author: Ben Pharn
 Ben Pharn
 ================================
 
-I am a first year Master's student here at Oregon State University.  I received my Bachelor of Science degree here as well.  I became a nuclear engineer in order to fulfill my dream of seeing a world powered by nuclear energy.  
+<img src="{{ site.url }}users/pharnb/images/image.jpg">
 
-Email: <a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a>
+I am a first year Master's student here at Oregon State University, having completed my undergraduate degree here as well.  I was born and raised in Southern California, where the majority of my family still resides.
+
+[Email] <a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)
+[Resume] (./files/resume.pdf)
+
+# Focus
+
+I dream of seeing a world powered by nuclear energy.  To that end, I have decided to focus on nuclear security and safeguards in order to help advance my dream
 
 # Research
 
-I work under Todd Palmer, and I am currently working on a project to help develop safeguards for small modular reactors. 
\ No newline at end of file
+I work under Todd Palmer, and I am currently working on a project to help develop safeguards for small modular reactors for Idaho National Laboratory.
+
+## Affiliations
+
+* American Nuclear Society: September 2012 - Present
+  * Member
+  * Local Chapter Webmaster September 2015 - June 2016
```

Comment:
this is me being nit picky.... OSU has an ANS student [section](http://students.ans.org/) not a chapter.